frontend/templates/product-list/index.tsx

```diff
@@ -55,10 +56,18 @@ export const getProductListServerSideProps = ({
    productListType,
 }: GetProductListServerSidePropsOptions): GetServerSideProps<ProductListTemplateProps> => {
    return async (context) => {
-      context.res.setHeader(
-         'Cache-Control',
-         'public, s-maxage=10, stale-while-revalidate=600'
-      );
+      if (context.query._vercel_no_cache) {
+         context.res.setHeader(
+            'Cache-Control',
+            'no-store, no-cache, must-revalidate'
```

Comment:
Why is it not enough to just have `no-store`? Doesn't setting `no-store` contradict `no-cache` and `must-revalidate`?

Reply:
It's not contradictory, its redundant. And its like this because from what people said on SO some browsers don't accept all of them so this should cover all our bases. In addition, because this is basically just for in house use, I don't have a problem with some slight header redundancy.